packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=T4Uehk3_wlY}
 ///
+///
+///{@tool sample --template=stateless_widget_scaffold_center.tmpl}
+///
+///In this example, the image is stretched to fill the entire [Container](https://master-api.flutter.dev/flutter/widgets/Container-class.html) which would not happen normally without using FittedBox.
+///
+///```dart
+///Widget build(BuildContext) {
+///  return Container(
+///    height: 400,
+///    width: 300,
+///    color: Colors.red,
+///    child: FittedBox(
+///      child: Image.network('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg'),
+///      fit: BoxFit.fill,
+///    ),
+///  );
+///}
+///```
+///
+///
+///{@end-tool}
+///
+///
```

Comment:
Remove extra line
```suggestion
```

Reply:
Thanks for the feedback, I have made the changes.

Reply:
I think you missed removing this extra line